includes/admin/class-wc-rest-payments-reporting-controller.php

```diff
@@ -5,7 +5,9 @@
  * @package WooCommerce\Payments\Admin
  */
 
+use WCPay\Core\Server\Request;
```

Comment:
⛏️ This is not used and can be removed.

```suggestion
```

Reply:
Thanks for spotting this. I have removed it here.